main.go

```diff
@@ -139,6 +147,14 @@ func runRecordings(conf *Config, camera *lepton3.Lepton3) error {
 	maxFrames := conf.MaxSecs * framesHz
 	numFrames := 0
 	lastFrame := 0
+
+	recordingLed := gpioreg.ByName(conf.LEDs.Recording)
+	if err := recordingLed.Out(gpio.Low); err != nil {
+		return fmt.Errorf("failed to set recording LED off: %v", err)
+	}
+	defer func() {
+		recordingLed.Out(gpio.Low)
+	}()
```

Comment:
This can just be `defer recordingLed.Out(gpio.Low)`. No need to wrap the call in another.